### PR TITLE
remove duplicate installs in dockerfile

### PR DESCRIPTION
**Pull request by frosty00**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -14,10 +14,3 @@ services:
     entrypoint: /bin/bash
     stdin_open: true
     tty: true
-    environment:
-      # NB: default values (they're blank here) — if you don't use the ${VAR:-default} syntax, Docker will be blaming about env vars not set..
-      - API_KEY="${API_KEY:-}"
-      - SECRET="${SECRET:-}"
-      - WALLET_ADDRESS="${WALLET_ADDRESS:-}"
-      - PRIVATE_KEY="${PRIVATE_KEY:-}"
```
Comment (xpl) [inline docker-compose.yml]: Those variables were originally introduced by a guy from TheOcean, I believe. They are used in some TheOcean-related examples. I decided to leave them temporarily, but I agree that there is no real need to put them there.

File: Dockerfile [hunk 2]
```diff
@@ -19,11 +19,11 @@ RUN apt-get -y install nodejs
 
 # Python 2
 RUN apt-get install -y python-pip
-RUN pip2 install --upgrade setuptools requests requests[security]
+RUN pip2 install --upgrade setuptools
 
 # Python 3
 RUN apt-get install -y python3 python3-pip
-RUN pip3 install --upgrade six setuptools aiohttp wheel requests pyopenssl tox twine
```
Comment (xpl) [inline Dockerfile]: Did you check that it still allows to run py2 & py3 tests without a problem? I copied these deps from our `.travis.yml` script but didn't have time to check whether there are some unneeded ones...

Comment (xpl) [reply Dockerfile]: @frosty00 Nah, it doesn't work. I've re-built the container image with those installs deleted, and it fails to execute even `npm run build`:

```
sh: 1: tox: not found
```

So please make sure that we have all the required dependencies installed to run those commands:

```
npm run build
```
```
node run-tests --js
```
```
node run-tests --python2
```
```
node run-tests --python3
```
```
node run-tests --php
```

Comment (xpl) [reply Dockerfile]: ...or, if you don't have the time to check it, we can simply keep those installs in the Dockerfile.

**Comment by frosty00** [comment]
`aiohttp` and `requests` are install automatically by doing the `python setup.py install`

**Comment by xpl** [comment]
@frosty00 Nice new userpic! Everything good, can merge now?

**Pull request merged**